src/arch/runtime/context_switching.cc

```diff
@@ -292,7 +289,7 @@ artificial_stack_t::artificial_stack_t(void (*initial_fun)(void), size_t _stack_
 #elif defined(__s390x__)
     sp -= 16; // r6-r13 and f8-f15.
 #elif defined (__powerpc64__)
-    sp += 24;
+    sp -= 20; // r14-r31, toc, cr.
```

Comment:
@mundaym Considering the factors performance and being light weight, I have not saved floating point and vector registers.There are no failures seen when non volatile floating point registers
are not saved and restored during context switch . I have tested on
ubuntu/RHEL powerpc64le systems. Also we can hit any bugs.